packages/service/src/api/x-sounds/x-sounds-controller.ts

```diff
@@ -26,19 +26,22 @@ class XSoundsController {
   async addXSound(
     @UploadedFile('soundFile') soundFile: any,
     @BodyParam('soundName') soundName: string,
+    @BodyParam('tags') tags: string,
```

Comment:
- IMO z punktu widzenia API to pole powinno być opcjonalne. 
- Dodaj do readme że ten endpoint ma nowe pole